include/beman/dump/dump.hpp

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef BEMAN_DUMP_DUMP_HPP_INCLUDED
+#define BEMAN_DUMP_DUMP_HPP_INCLUDED
+
+#include <array>
+#include <print>
+#include <string_view>
+
+namespace beman::dump {
+    namespace detail {
+        template <std::size_t N>
+        constexpr auto format_string = []{
```

Comment:
A documentation comment for this variable template would be nice.

Reply:
Temporary TODO comment added